src/foam/nanos/notification/email/services.jrl

```diff
@@ -28,8 +28,7 @@ p({
       .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
       .setJournalName("emailMessages")
       .setFixedSize(new foam.dao.FixedSizeDAO.Builder(x)
-        .setComparator(foam.mlang.MLang.DESC(foam.nanos.notification.email.EmailMessage.CREATED))
-        .setSize(10000)
+        .setPurgeSize(10000)
```

Comment:
please re-add the comparator and skip and size.
and the purgepercent default of 10 is fine.